.circleci/config.yml

```diff
@@ -0,0 +1,129 @@
+version: 2.1
+
+workflows:
+  build_and_test:
+    jobs:
+      - build
+      - jenkins_test:
+          jenkins_version: lts
+
+  nightly_jenkins_test:
+    triggers:
+      - schedule:
+          cron: "0 6 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - jenkins_test:
+          jenkins_version: lts
+      - jenkins_test:
+          jenkins_version: latest
+
+jobs:
+  build:
+    docker:
+      - image: circleci/openjdk:8u222-jdk-stretch
+
+    working_directory: ~/repo
+
+    environment:
+      # Customize the JVM maximum heap limit
+      MAVEN_OPTS: -Xmx3200m
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "pom.xml" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
```

Comment:
Can`t find where you save this cache.

Reply:
It's actually saved later, check `save_cache` step. We need to restore the cache (if it's existing) before collecting dependencies and saving the cache.